src/app/cluster/details/cluster/add-machine-network/component.spec.ts

```diff
@@ -29,13 +29,21 @@ import {NotificationService} from '@core/services/notification';
 import {SharedModule} from '@shared/module';
 import {AddMachineNetworkComponent} from './component';
 
+const modules: (BrowserModule | RouterTestingModule)[] = [
```

Comment:
You do not list all types there. You can:
1. Skip the const as it is used in once place only (preferred).
2. Remove type completely.